akka-actor-tests/src/test/scala/akka/io/TcpIntegrationSpecSupport.scala

```diff
@@ -9,7 +9,7 @@ import scala.collection.immutable
 import akka.testkit.{ AkkaSpec, TestProbe }
 import akka.actor.ActorRef
 import akka.io.Inet.SocketOption
-import akka.testkit.SocketUtil._
+import akka.testkit.SocketUtil.temporaryServerAddress
```

Comment:
@johanandren I think prefer not to do wildcard imports for merge reasons?

Reply:
They are allowed, but their scope changed: in 2.13, imports take precedence over objects defined in the same package. I think `SocketUtil._` brought some class into scope that overlapped with a class in the current package. Rather than excluding that class, I thought it'd be safer/clearer to make the import more specific.

Reply:
Ah, thanks for explaining. Sounds good to me.